### PR TITLE
feat(cache-client): update test login endpoint

### PR DESCRIPTION
**Pull request by Harasz**

### CODE REVIEW

File: src/modules/cacheClient/cacheClient.service.ts [hunk 1]
```diff
@@ -319,8 +319,10 @@ export class CacheClient implements ICacheClient {
      */
     private async isAuthenticated(): Promise<boolean> {
         try {
-            await this.httpClient.get<Asset[]>(`${TEST_LOGIN_ENDPOINT}${this._signerService.did}`);
-            return true;
+            const { data } = await this.httpClient.get<{
```
Comment (JGiter) [inline src/modules/cacheClient/cacheClient.service.ts:322]: I guess it should return user for whom access token was issued

**Review by JGiter — CHANGES_REQUESTED** [review]

File: src/modules/cacheClient/cacheClient.service.ts (hunk 1 above)
Comment (jrhender) [reply src/modules/cacheClient/cacheClient.service.ts:322]: Hmm, IMO, a name that starts with "is" communicates that the return value is a boolean. Personally, I think the currently implementation is good.

File: src/modules/cacheClient/cacheClient.service.ts [hunk 2]
```diff
@@ -319,8 +319,10 @@ export class CacheClient implements ICacheClient {
...
+                user: string | null;
+            }>(`${TEST_LOGIN_ENDPOINT}`);
+            return data.user === this._signerService.did;
```
Comment (jrhender) [inline src/modules/cacheClient/cacheClient.service.ts:325]: In the unlikely event that `this._signerService.did` was `null`, I wouldn't want `isAuthenticated` to return `true`
```suggestion
            return data.user ? data.user === this._signerService.did : false;
```

**Review by jrhender — APPROVED** [review]
A small suggestion, but other than that I think it looks good

**Pull request merged**